nautobot/extras/templates/extras/job_detail.html

```diff
@@ -284,6 +284,32 @@
                     {% endif %}
                 </td>
             </tr>
+            <tr>
+                <td>Default Job Queue</td>
+                <td>
+                    {% if object.default_job_queue %}
+                        {{ object.default_job_queue|hyperlinked_object }}
+                    {% else %}
+                        {{ None|placeholder }}
+                    {% endif %}
```

Comment:
Pretty sure this can just be reduced to `{{ object.default_job_queue|hyperlinked_object }}` without the if/else.